packages/remediations/src/common/SystemsTable.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Provider } from 'react-redux';
+import propTypes from 'prop-types';
+import promiseMiddleware from 'redux-promise-middleware';
+import ReducerRegistry from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
+import InventoryTable from '@redhat-cloud-services/frontend-components/Inventory/InventoryTable';
+import {
+    fetchSystemsInfo,
+    inventoryEntitiesReducer as entitiesReducer
+} from '../utils';
+
+const SystemsTable = ({ registry, allSystemsNamed, allSystems, hasCheckbox, disabledColumns, bulkSelect }) => {
+
+    const inventory = useRef(null);
+
+    const inventoryApi = useRef({});
+
+    const onRefresh = (options, callback) => {
+        if (!callback && inventory && inventory.current) {
+            inventory.current.onRefreshData(options);
+        } else if (callback) {
+            callback(options);
+        }
+    };
+
+    return (
+        <InventoryTable
+            hideFilters={{
+                tags: true,
+                registeredWith: true,
+                stale: true
+            }}
+            columns={(columns) => columns.filter(column => !disabledColumns.includes(column.key)) }
+            noDetail
+            variant="compact"
+            hasCheckbox={hasCheckbox}
+            showTags
+            bulkSelect={bulkSelect}
+            onRefresh={onRefresh}
+            ref={inventory}
+            getEntities={(_i, config) => fetchSystemsInfo(config, allSystemsNamed, inventoryApi.current)}
```

Comment:
you can get the inventoryApi via https://github.com/RedHatInsights/frontend-components/blob/master/packages/inventory/doc/custom_fetch.md#defaultgetentities